SR2019/2019-10-16-soton-kickstart-deny.md

```diff
@@ -0,0 +1,13 @@
+---
+to: Student Robotics 2019, Southampton Segment
+subject: The London kickstart is full
+note: This email is to be sent to those who are _not_ going to the London kickstart.
+---
+
+Hello!
+
+We regret to update you that the London kickstart venue is full, and so your team will not be able to go.
```

Comment:
Perhaps keep this positive? While it's clear from the following paragraph that there is a space elsewhere, this first paragraph sounds almost final that they can't come to a Kickstart.

How about something like:
> We regret to inform you that the London kickstart venue is full. There is space for you at the Southampton venue and we look forward to seeing you there instead.
>
> If you are unable to get to Southampton, then we can arrange another way to get you your kit.